lemur/certificates/schemas.py

```diff
@@ -353,6 +355,20 @@ def handle_subject_details(self, data):
                 data.pop("organization", None)
                 data.pop("organizational_unit", None)
 
+    @post_dump
+    def handle_certificate(self, cert):
+        if cert['root_authority']:
+            authority = cert['root_authority']
+        else:
+            authority = cert['authority']
+        authority_id = authority['id']
+        plugin = plugins.get(authority['plugin']['slug'])
+        cert['body'], cert['csr'], cert['chain'] = plugin.wrap_certificate(
```

Comment:
should this be: 

```
    body, csr, body = plugin.wrap_certificate
```

Reply:
> sorry if this taking some time. I am trying to understand what is going in the background.

It's a normal process, I expected it before doing PR.

> sorry if I might be missing something obvious with the model, or how the schema works, but how come these two are different?

Don't forget the informations I'm looking for:

- the plugin name to use
- the ID of the authority to find it's private key

This two informations are in Authorities table.

We have the contexts:

- "I get the informations for a certificate" in this case the plugin and the ID are in cert['authority'], cert['root_authority'] is None
- "I get the informations for an authority", in this case the plugin and the ID are in cert['root_authority'], cert['authority'] is None

This is why I take the informations either in one of the variables or in the other.